apps/condo/bin/lib/billing-context-script-core.js

```diff
@@ -0,0 +1,130 @@
+const dayjs = require('dayjs')
+const { get, isEmpty } = require('lodash')
+
+const { COUNTRIES, DEFAULT_LOCALE } = require('@condo/domains/common/constants/countries')
+
+const { BillingIntegrationOrganizationContext } = require('@condo/domains/billing/utils/serverSchema')
+
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
+
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const { EXEC_COMMAND, BASE_NAME, MAX_SEND_COUNT, FORCE_SEND, DATE_FORMAT } = require('./constants')
+const { getMonthStart } = require('./helpers')
+const { ScriptCore, runFnWithArgs } = require('./script-core')
+
+class BillingContextScriptCore extends ScriptCore {
```

Comment:
don't use inheritance. It does not improve the readability or understandability in the case